swagger/swagger.json

```diff
@@ -7602,7 +7602,7 @@
         ],
         "properties": {
           "xml_dir": {
-            "default": "/mnt/code/src/emqx/0/_build/emqx/rel/emqx/etc/lwm2m_xml",
+            "default": "/home/thales/dev/emqx/emqx/_build/emqx/rel/emqx/etc/lwm2m_xml",
```

Comment:
Oops, looks like this one takes some paths from the local machine... :see_no_evil: